contracts/tips/README.md

```diff
@@ -0,0 +1,192 @@
+# Tips contract
+
+Contract that allows sending tips between users as well tracking sent and received tips for each one of them.  
+To easily interact with the contract you can use the `tips` script available [here](https://github.com/desmos-labs/contract-utils/tree/main/utils), 
+otherwise you can take a look at the supported messages in the following sections.
+
+## Instantiate Message
+Allows to initialize the contract. This message has the following parameters:
+* `admin`: Address of the user that controls the contract;
+* `subspace_id`: Application which is deploying the contract;
+* `service_fee`: Fee that the users need to pay to use the contract, can be percentage, fixed or 
+`null` to signal that the contract shouldn't collect fees;
+* `tips_history_size`: Number of records saved of a user tips history, can be also `0` to signal that the contract
+shouldn't save any tip history.
+
+An example of instantiate message with a percentage fee
+```json
+{
+  "admin": "desmos1.....",
+  "subspace_id": "1",
+  "service_fee": {
+    "percentage": {
+      "value": "0.1"
+    }
+  },
+  "tips_history_size": 10
+}
+```
+If you prefer to have a fixed fee instead you can replace the `service_fee` object with something like this
+```json
+{
+  "fixed": {
+    "amount": [
+      {
+        "amount": "1000",
+        "denom": "udsm"
+      }
+    ]
+  }
+}
+```
+
+## Execute Messages
+
+### SendTip
+Allows to send a tips to a user or to the author of a post. This message has the following parameters:
+* `amount`: Tip amount;
+* `target`: Who receives the tip, can be a user address or a post id to signal appreciation for a post created from a user.  
```

Comment:
```suggestion
* `target`: Tip target: it can be either a user address or a post id to signal appreciation towards a specific content rather than a user.  
```